app_data/communities.yaml

```diff
@@ -0,0 +1,98 @@
+- access:
+    visibility: public
+  slug: biosyslit
+  logo: biosyslit.png
+  metadata:
+    title: Biodiversity Literature Repository
+    description: A community to share publications related to bio-systematics. The goal is to provide 1. open access to publications cited in publications or in combination with scientific names 2. open access FAIR data with focus on taxon treatments and figures liberated from publications 3. a digital object identifier (DOI) to enable citation of the publications and data including direct access to its digital representation. For additional search functionality can be used. This includes also searches in CrossRef, DataCite, PubMed, RefBank, GNUB and Mendeley.
```

Comment:
**Note**: fields `description`, `curation_policy `and `page` were added as plain text and currently look ugly in UI.
Originally, these fields consist of HTML, which currently cannot be parsed at the community details page.
See [the issue](https://github.com/zenodo/zenodo-rdm/issues/65) created for fixing HTML parsing on UI side.